Vic2ToHoI4/Data_Files/ideologicalDecisions.txt

```diff
@@ -247,6 +269,41 @@ fascism_on_the_rise = {
 			factor = 0
 		}
 	}
+
+	ignite_the_fascism_civil_war_single_state = {
+		icon = generic_ignite_civil_war
+		name = ignite_the_fascism_civil_war
+
+		available = {
+			has_stability < 0.5
+			fascism > 0.50
+		}
+
+		visible = {
+			has_country_flag = preparation_for_fascism_civil_war
+			OR = {
+				has_idea_with_trait = fascism_demagogue
+				has_idea_with_trait = syncretic_revanchist
+			}
+			NOT = { has_government = fascism }
+			check_variable = { num_owned_states = 1 }
```

Comment:
I don't know how variables work. Please, be sure "num_owned_state = 1" returns false if more than one states are owned

Reply:
This is borrowed from the vanilla files, so presumably works correctly.